manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,17 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+The compiler is able to emit some information on its internal stages.
+It can output ".cmt" files for the implementation of the compilation unit and
+".cmti" for signatures. This operation is activated by passing the key
+"-bin-annot" to the compiler (see the description of "-bin-annot" below).
```

Comment:
The manual uses `option` rather than `key`.  The sentence feels a bit redundant, maybe it could be merged with the sentence above?